include/picongpu/fields/Fields.def

```diff
@@ -43,6 +47,8 @@ namespace picongpu
             std::stringstream str;
             str << T_Species::FrameType::getName();
             str << "_";
+            str << T_Filter::getName();
+            str << "_";
```

Comment:
Is it in the middle for a reason? I have no idea what's better, I guess what's consistent with other namings

Reply:
I thought that this would be better since the filter applies to the particles and not the field